src/main/java/org/myrobotlab/swing/ProgramABGui.java

```diff
@@ -81,6 +84,11 @@ public ProgramABGui(String boundServiceName, SwingGui myService) {
     filter.setSelected(true);
 
     response.setAutoscrolls(true);
+    response.setEditable(false);
+    response.setEditorKit(responseKit);
+    response.setDocument(responseDoc);
+
+    responseKit.insertHTML(responseDoc, responseDoc.getLength(), "<font face=\\\"Verdana\\\">Conversation :<br/>", 0, 0, null);

```

Comment:
heh, sorry - found another line :P

Reply:
Then I don't think it's working the way you want it to.
I just tested out this:
```
responseKit.insertHTML(responseDoc, responseDoc.getLength(), "<font face=\\\"Verdana\\\">Conversation :<br/>", 0, 0, null);
responseKit.insertHTML(responseDoc, responseDoc.getLength(), "<i><b>" + "botname" + "</b>: " + "text" + "</i>", 0, 0, null);
```
And the resulting html markup is:
```
<html>
  <head>
    
  </head>
  <body>
    <font face="\Verdana\">Conversation :<br></font>
<i><b>botname</b>: text</i>
  </body>
</html>
```